client/components/Nav.jsx

```diff
@@ -20,23 +20,20 @@ function Nav() {
   }
 
   return (
-    <>
-      <NavGroup>
-        <NavLink to="/">Home</NavLink>
-        <IfAuthenticated>
-          <NavLink to="/" onClick={handleLogOff}>
-            Log off
-          </NavLink>
-          <p>{user.username}</p>
-        </IfAuthenticated>
-        <IfNotAuthenticated>
-          <NavLink to="/" onClick={handleSignIn}>
-            Sign In
-          </NavLink>
-        </IfNotAuthenticated>
-      </NavGroup>
-      <h1>Fruit FTW!</h1>
-    </>
+    <div>
+      <NavLink to="/">Home</NavLink>
+      <IfAuthenticated>
+        <NavLink to="/" onClick={handleLogOff}>
```

Comment:
This shouldnt be a NavLink as you need `handleLogOff` to happen before the page Navigates. There is a change you would navigate to the new page before the log off finishes

Reply:
Should we change this to a `Button` with no `to='/'` and React-Router-Dom's `useNavigate` navigate("/") inside the `handleLogOff`?

@ZinHoang @pei-yeshuang-zhang This looks like something for the front-end team to consider while creating the Nav.

Reply:
Yep thats exactly how you should do it